src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesInformation.cs

```diff
@@ -15,32 +15,32 @@
 // </copyright>
 
 using System;
+using System.Diagnostics;
 #if NET452
 using OpenTelemetry.Internal;
 #endif
-using OpenTelemetry.Trace.Export;
 
 namespace OpenTelemetry.Exporter.ZPages.Implementation
 {
     /// <summary>
-    /// Stores the span information aggregated according to span name.
+    /// Stores the activity information aggregated according to activity name.
     /// </summary>
-    public class ZPagesSpanInformation
+    public class ZPagesInformation
```

Comment:
Same feedback on the `ZPagesInformation` name. I think it should also be `internal`. But would it also be better with a different name? `ZPagesActivityAggregate`? 🤷‍♂️